src/models/darp10/README.md

```diff
@@ -0,0 +1,67 @@
+# Darter Pro (darp10 & darp10-b)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Darter Pro](./img/darp10.webp)
+
+The System76 Darter Pro is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel Core Ultra 1st generation (Meteor Lake) CPUs
+        - [Core Ultra 7-155H](https://ark.intel.com/content/www/us/en/ark/products/236847/intel-core-ultra-7-processor-155h-24m-cache-up-to-4-80-ghz.html)
+        - [Core Ultra 5-125H](https://ark.intel.com/content/www/us/en/ark/products/236848/intel-core-ultra-5-processor-125h-18m-cache-up-to-4-50-ghz.html)
+- BIOS
+    - XMC XM25QU256CXIQT08S flash chip
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU: Intel Arc Graphics
+    - eDP display:
+        - 16" 1920x1200@60Hz LCD (darp10)
+            - LCD panel: AUO B160UAN03.0 (or equivalent)
+                - Brightness: 250 nits (cd/m²)
+                - sRGB color coverage: 65%
+        - 14" 1920x1200@60Hz LCD (darp10-b)
+            - LCD panel: Innolux N140JCA-ELL (or equivalent)
+                - Brightness: 300 nits (cd/m²)
+                - sRGB color coverage: 66%
+    - External video outputs:
+        - 1x HDMI 2.1
+        - 1x DisplayPort 2.1 over USB-C
+- Memory
+    - Up to 96GB (2x48GB) dual-channel DDR5 SO-DIMMs @ 5600 MHz
+- Networking
+    - Gigabit Ethernet
+    - M.2 PCIe/CNVi WiFi/Bluetooth
+        - Intel Wi-Fi 6E AX210/AX211
+            - Bluetooth 5.3
+- Power
+    - 90W (19V, 4.74A) DC-in port
+        - Barrel size: 5.5mm (outer), 2.5mm (inner)
+    - Included AC adapter: Chicony A16-090P1A
+        - AC power cord type: IEC C5
+    - USB-C charging compatible with 90W+ charger
+    - 73Wh 4-cell Lithium-ion battery
+        - Model number: L140BAT-4
```

Comment:
I believe the battery model number is `V540-BAT-4-73` now. Should be the same on both sizes.

Reply:
Updated.